README.md

```diff
@@ -18,7 +18,13 @@ To keep track of the changes between releases check the [changelog](@ref Changel
 
 # Features {#Features}
 
-Brion provides the following major features:
+Brion provides two libraries: one for low level access to the data model and
```

Comment:
I don't know what happened here. I'm pretty sure I addressed this and all the remaining comments yesterday and I doubt I didn't push. I'll check later because I don't have the laptop with me at the moment.